README.md

```diff
@@ -29,6 +29,50 @@ and have it be rendered properly at `2019`.
 ### Videos
 
 An even more fragile process which needs documenting and fixing.
+
+WIP Docs on how to do this:
+
+1. In `_data/<year>.<city>.speakers.yaml`, add a `youtubeId: 12345678901` key value pair to each talk.
+
+2. Make sure the directory `videos/<city>/<year>` is included in the Video Archive `toctree` in `docs/videos/index.rst`.
+
+3. In `conf.py`, around line 130 uncomment the following two lines:
+
+   ```
+   #from _ext.videos import main
+   #html_context.update(main())
+   ```
```

Comment:
We can definitely make this better --  we could even have it depend on the same logic around `BUILD_VIDEOS` to make this a simpler step.

Reply:
:100:  was thinking that after the recent convo around videos